lib/routes/emails.js

```diff
@@ -689,8 +689,17 @@ module.exports = (log, db, mailer, config, customs, push) => {
           })
           .then(deleteEmail)
           .then(() => {
+            const resolvedEmail = account.emails.filter((item) => {
+              if (item.normalizedEmail === email.toLowerCase() && item.isVerified) {
+                return item
+              }
+            })
```

Comment:
Can you please add a comment to this block to explain what it's doing?  If I didn't have the name of the issue to guide me, I would have puzzled over it for a long time.  Something as simple as "Don't bother sending a notification if removing an email that was never verified" would make it much clearer for future readers.

Reply:
Alternately, you may be able to make it clearer with careful naming, using something like:

```
const emailWasVerified = account.emails.find(..)
if (! emailWasVerified) { return P.resolve() }
```